azure_arc_servers_jumpstart/automanage/artifacts/arc_automanage.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Environment Variables
+export automanageAccountName='<Name of your Azure Automanage Account>'
+export location='<Azure Region where your Azure Arc enabled server is registered>'
+export resource_group='<Azure Resource Group where your Azure Arc enabled server is registered>'
```

Comment:
```suggestion
export resourceGroup='<Azure Resource Group where your Azure Arc enabled server is registered>'
```

Also make sure to update where relevant

Reply:
changed